src/styles/main.scss

```diff
@@ -1,3 +1,43 @@
+$snakeItemColors: (
+  #f00: 1,
+  #c00: 2,
+  #900: 3,
+  #600: 4,
+  #300: 5,
+  #000: 6,
+);
+
+html {
+  font-family: Arial, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.snake {
+  display: grid;
+  grid-template: repeat(2, 1fr) / repeat(3, 1fr);
+
+  @media (max-width: 899px) {
+    grid-template: repeat(3 ,1fr) / repeat(2, 1fr);
+  }
+
+  @media (max-width: 599px) {
+    grid-template: repeat(6, 1fr) / 1fr;
+  }
+
+  &__item {
```

Comment:
The width of each block should be not less than its height (300px) https://prntscr.com/tgwtx9